kubernetes/ansible-kube/environments/kube/files/invoker.yml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: invoker
+  namespace: openwhisk
+  labels:
+    name: invoker
+spec:
+  replicas: 1
+  serviceName: "invoker"
+  template:
+    metadata:
+      labels:
+        name: invoker
+    spec:
+      restartPolicy: Always
+
+      volumes:
+      - name: cgroup
+        hostPath:
+          path: "/sys/fs/cgroup"
+      - name: runc
+        hostPath:
+          path: "/run/runc"
+      - name: dockerrootdir
+        hostPath:
+          path: "/var/lib/docker/containers"
+      - name: dockersock
+        hostPath:
+          path: "/var/run/docker.sock"
+      - name: apparmor
+        hostPath:
+          path: "/usr/lib/x86_64-linux-gnu/libapparmor.so.1"
+
+      containers:
+      - name: invoker
+        imagePullPolicy: IfNotPresent
+        image: danlavine/whisk_kube_invoker
```

Comment:
The issue with this is that each invoker needs to know which index they are. In the Kube world, I am doing this by using a couple of things. The Invokers are deployed via [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) which give each instance a static hostname of `invoker-X`. From here I have a custom docker file for the [invoker](https://github.com/DanLavine/openwhisk-devtools/blob/kube/kubernetes/docker/invoker/Dockerfile) which uses this hostname when starting the invoker instance [here](https://github.com/DanLavine/openwhisk-devtools/blob/kube/kubernetes/docker/invoker/invoker_start_script.sh).